controllers/openstack_ansibleee_controller.go

```diff
@@ -273,7 +273,8 @@ func (r *OpenStackAnsibleEEReconciler) jobForOpenStackAnsibleEE(instance *redhat
 					Labels:      ls,
 				},
 				Spec: corev1.PodSpec{
-					RestartPolicy: corev1.RestartPolicy(instance.Spec.RestartPolicy),
+					RestartPolicy:      corev1.RestartPolicy(instance.Spec.RestartPolicy),
+					ServiceAccountName: "openstack-ansibleee-operator-openstack-ansibleee",
```

Comment:
I admit to not fully understanding how this all works, but what is the point of creating the openstack-ansibleee ServiceAccount, but not use it here?

Reply:
I tried to follow the other repo's service account naming policy: $reponame-$servicename (e.g. `keystone-operator-keystone`, `mariadb-operator-mariadb`....) 

I added an extra "openstack" here, let me remove it.